tasks/create_target_vm/03_hosted_engine_final_tasks.yml

```diff
@@ -248,6 +248,13 @@
     become_user: vdsm
     become_method: sudo
     changed_when: true
+  - name: Verify copy of VM disk
+    command: qemu-img compare {{ local_vm_disk_path }} {{ he_virtio_disk_path }}
+    environment: "{{ he_cmd_lang }}"
+    become: true
+    become_user: vdsm
+    become_method: sudo
+    changed_when: true
   - name: Remove temporary entry in /etc/hosts for the local VM
```

Comment:
This is not needed normally, but can help to debug issues.

Do we have a way to enable it only in --debug mode?

If not I think we should not add this since it slows down the instillation.

Reply:
What happen when the comparison fails ? 
maybe add an error message for the user.

Additionally, I would add a variable to enable/disable this check because of what Nir said and add document it in the README

Reply:
If the check fail we need to fail the installation, this means something went wrong with the copy, or storage is faulty.

It think this will never happen, but with gluster thieving is possible.

Reply:
Added `he_debug_mode` variable to execute debug tasks